### ISSUE TITLE
Add documentation on formula escaping

### ISSUE THREAD

**Issue by gor181** [datacamp/authoring#65]
@vincentvankrunkelsven commented on [Fri Jun 16 2017](https://github.com/datacamp/teach-app/issues/290)

LaTeX formulas in markdown should be escaped. E.g. in $x\_{1}+x\_{3}$, the `_` must be escaped: $x\\\_{1}+x\\\_{3}$

**Comment by ncarchedi** [comment]
@gor181 is there a more general source for this information or is it specific to the way teach does things?

**Comment by ncarchedi** [comment]
@richierocks would you like this documented anywhere in particular in the authoring docs?

**Comment by gor181** [comment]
Source is @vincentvankrunkelsven memory 😃. Another source would be checking the teach-parser source code.

From what I know escaping markdown is the only necessary thing to get them working properly.

**Comment by richierocks** [comment]
@ncarchedi Can adding equations by productized? In the meantime, a page in the 'Writing NormalExercises' directory would be most natural, I think.

https://github.com/datacamp/authoring/tree/master/docs/courses/exercises/normal-exercises

**Comment by ncarchedi** [comment]
@richierocks I don't think we want to create a whole UI just for building equations given the long list of other stuff we need to get to. But we could definitely create a small markdown formula cheat sheet and link to it somewhere in the editor or in a help article so it's easily findable. WDYT?

**Comment by ncarchedi** [comment]
I'd rather people don't have to leave the editor to find it.
